streamer/src/tls.rs

```diff
@@ -0,0 +1,268 @@
+use {
```

Comment:
I like the boilerplate code is isolated now. The only think is that I would like to see maybe a bit more documentation on this module side. Like:

```
//! This module is to use ed25519_dalek which is known to be better for this and that reason
```